app/models/concerns/github_importable.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+module GitHubImportable
```

Comment:
I agree, I don't think it should be a model concern. Any suggestions as to where this should go?